receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper.go

```diff
@@ -288,7 +297,7 @@ func (s *scraper) scrapeAndAppendMemoryUtilizationMetric(now pcommon.Timestamp,
 }
 
 func (s *scraper) scrapeAndAppendDiskMetrics(now pcommon.Timestamp, handle processHandle) error {
-	if !(s.config.MetricsBuilderConfig.Metrics.ProcessDiskIo.Enabled || s.config.MetricsBuilderConfig.Metrics.ProcessDiskOperations.Enabled) || runtime.GOOS == "darwin" {
+	if !(s.config.MetricsBuilderConfig.Metrics.ProcessDiskIo.Enabled || s.config.MetricsBuilderConfig.Metrics.ProcessDiskOperations.Enabled) || s.config.MuteProcessIOError || runtime.GOOS == "darwin" {
```

Comment:
Is this an intended change? So we don't even attempt to scrape disk process metrics if `mute_process_io_error=true`?

If so, why do we need another configuration option just to disable collection disk process metrics?
Users should be able to achieve the same with:
```
metrics:
  process.disk.io:
    enabled: false
  process.disk.operations:
    enabled: false
```

Reply:
Oh, I meant to mute these errors, correction in the next commit
```
io, err := handle.IOCounters()
if err != nil {
	if s.config.MuteProcessIOError {
		return nil
        }
	return err
}
```